doc/user/bgp.rst

```diff
@@ -1668,6 +1671,16 @@ address-family:
 
    Disables import or export of routes betwen the current unicast VRF and VPN.
 
+.. index:: import vrf VRFNAME
+.. clicmd:: import vrf VRFNAME
+
+   Enables direct importation of VRF routes to another VRF.  The RD and RT
+   are auto derived and are not needed for VRF - VRF route leaking.
```

Comment:
Rather than this text, I propose the following:

> Shortcut syntax for specifying leaking from vrf VRFNAME to the current VRF using the VPN RIB as intermediary. The RD and RT are auto derived and should not be specified explicitly for either the source or destination VRFs.
> 
> This shortcut syntax mode is not compatible with explicit "import vpn" and "export vpn" statements for the two VRFs involved. The CLI will disallow attempts to configure incompatible leaking modes.

Reply:
done.